src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,35 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector
+            = Injector.getInstance("mate.academy.dao");
```

Comment:
The package name "mate.academy.dao" used for initializing the `Injector` might be incorrect. Please verify that this package name is correct to avoid potential runtime errors.